peps/pep-0735.rst

```diff
@@ -0,0 +1,105 @@
+PEP: 735
+Title: Enabling certificate verification by default for stdlib mail modules
+Author: Martin Schobert <martin@pentagrid.ch>
+Sponsor: Victor Stinner <vstinner@python.org>
```

Comment:
I would prefer that you ask me to be a sponsor, and ask me to review the PEP before you go ahead and propose it directly.

```suggestion
Author: Martin Schobert <martin@pentagrid.ch>
```

Reply:
Context: I offered to sponsor a PEP if someone wants to write it: https://github.com/python/cpython/issues/91826#issuecomment-1810023714

Reply:
Thanks for the background.

It's great to be proactive and write the PEP, but it's also very important to have a sponsor beforehand who's happy with the text, and also to help with the process. [PEP 1](https://peps.python.org/pep-0001/#submitting-a-pep) says:

> Once the sponsor or the core developer(s) co-authoring the PEP deem the PEP ready for submission, the proposal should be submitted as a draft PEP via a [GitHub pull request](https://github.com/python/peps/pulls).

I suggest we close this PR for now, and @nitram2342 contacts @vstinner privately and they decide how to proceed. If they can agree on a text together, then update and re-open this PR, or a fresh one, and we can take it from there.

How does that sound?

Reply:
If someone wants me to sponsor a PEP, I would prefer to read it and approve it ahead, right.

@nitram2342: If you would like to sponsor your PEP, I suggest you closing this PR, as @hugovk suggests.

Reply:
Okay, that is fine. I will do it this way.